src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -183,8 +182,19 @@ def create_query(
         return "\n".join(query)
 
 
+def _is_signature_type_a_string(sig_type: str) -> bool:
+    return sig_type.lower() in ["string", "varchar"]
```

Comment:
int and number don't need thit check?

Reply:
What with boolean?

Reply:
Works out of the box, I've added a test for that